src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/CryptoStream.cs

```diff
@@ -717,7 +717,9 @@ private async Task CopyToAsyncInternal(Stream destination, int bufferSize, Cance
                 pinHandle.Free();
             }
             ArrayPool<byte>.Shared.Return(rentedBuffer);
+#pragma warning disable IDE0059
             rentedBuffer = null;
+#pragma warning restore IDE0059
```

Comment:
Why is this needed?  The state machine should be nulled out when the async method completes, so we shouldn't need the nulling out at the end to ensure the field is cleared.

Reply:
Removed